actors-library/src/main/java/com/truecaller/androidactors/MessageBase.java

```diff
@@ -42,4 +43,33 @@ protected MessageBase(@NonNull ActorInvokeException exception) {
     public ActorInvokeException exception() {
         return mExceptionTemplate;
     }
+
+    @NonNull
+    protected static String logParam(Object parameter, int level) {
+        if (level == SecureParameter.LEVEL_NO_INFO) {
+            return "<value>";
+        }
+
+        if (parameter == null) {
+            return "null";
+        }
+
+        if (parameter instanceof String) {
```

Comment:
Maybe should check for `CharSequence` instead of `String`?

Reply:
Done